src/templates/components/forms_common_rows.html.tt

```diff
@@ -44,6 +44,12 @@
         hint=hint || (no_hint ? '' : l('Kindly provide your complete address.<br>This will be used to authenticate your identity when you open a real account.')) %]
 [% END %]
 
+[% BLOCK address_validation_text %]
+    <span class="gr-12">
+        <p>[% l("You can use letters and numbers (A-Z, a-z, 0-9) and the following special characters: -.'#;:()") %]</p>
```

Comment:
since we're not using accordions anymore, can we just add this as the error message for address field instead of the old message? or there is some other styling difference that I'm not aware of?